.github/workflows/jsdom-ci.yml

```diff
@@ -29,29 +29,21 @@ jobs:
     env:
       TEST_SUITE: 'node-canvas'
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - node-version: '20'
-            canvas-version: '^2.11.2'
-          - node-version: '22'
-            canvas-version: '^3.0.0-rc3'
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: 'recursive'
       - uses: actions/setup-node@v4
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: '22'
       - name: Install required image manipulation packages with APT
         run: sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
       - name: Setup HOSTS file for Web Platform Test server
         run: ./test/web-platform-tests/tests/wpt make-hosts-file | sudo tee -a /etc/hosts
       - name: Install dependencies
-        run: npm ci && npm install canvas@${{ matrix.canvas-version }}
+        run: npm ci
       - name: Run tests
-        run: npm test -- --retries 1
+        run: npm i canvas && npm test -- --retries 1
```

Comment:
I know that the test setup is not that great (probably there should be a test project with both canvas and jsdom) but:
1. Installing dependencies should not be done in 2 steps. `canvas` should be installed together with the other dependencies. This will also keep the test output clean.
2. Not specifying a version when installing a package will always install the latest. This will install `canvas@4.0.0` when it will be released.

Reply:
I'm currently on my way to a ski trip and will implement the changes this evening.

I have a question regarding your second point: will `npm i canvas` always install the latest version even when the field `"peerDependencies": { "canvas": "^3.0.0" }` is set? I thought (or hoped) it would install the latest minor or patch version of the v3 major version.

Reply:
`npm i canvas` will modify `package.json` and the lockfile and will add the latest version of canvas to the `dependencies` group. I'm not sure if npm will also remove it from `peerDependencies` (might also depend on the npm version), some package managers do this.

_Viel Spaß! :)_

Reply:
Moved the `npm i canvas@3` to the same line as `npm ci` and specified canvas in its installtion with `@3`